.github/workflows/nightly.yml

```diff
@@ -233,9 +63,8 @@ jobs:
           time=$(git show --no-patch --format=%ad --date=format:%H%M%S)
 
           # Update the version to a nightly one
-          # This sets the nightly version to something like `1.20231117.1130416`
-          # The 1 at the beginning of the patch number is just so it never starts with zero
```

Comment:
Is there any reason for deleting this comment? I think it's important because it reminds us why there is a 1 at the beginning of the patch, otherwise we might end up removing it which will break the job for commits whose hour starts with `0`.